appveyor.yml

```diff
@@ -27,6 +27,11 @@
               BOND_VS_NUM: 14
               BOND_ARCH: 32
               BOND_BOOST: 60
+            - BOND_BUILD: Python
+              BOND_VS: "Visual Studio 14 2015"
+              BOND_VS_NUM: 14
+              BOND_ARCH: 64
+              BOND_BOOST: 63
```

Comment:
This build failed in AppVeyor: https://ci.appveyor.com/project/sapek/bond/build/1.0.1272/job/sp5n8a3a06id2twy